packages/language/src/language-server/code-actions/apply-quick-fixes.ts

```diff
@@ -82,25 +82,58 @@ export async function quickFixResolveInclude(
     diagnostics: [diagnostic],
     command: {
       title: "Apply INCLUDE fix",
-      command: "pli.applyIncludeFix",
+      command: "pli.applyQuickFixResolveInclude",
       arguments: [procGrpsFileUri.toString(), newContent],
     },
   };
 
   return action;
 }
 
+export async function quickFixCreateConfig(
+  diagnostic: Diagnostic,
+): Promise<CodeAction | undefined> {
+  const entryUri = UriUtils.basename(URI.parse(diagnostic.data.entryUri));
+  if (!entryUri) {
+    return;
+  }
+  const action: CodeAction = {
+    title: `Create a '.pliplugin' folder in the project root using the current file as the entry point in 'pgm_conf.json'`,
```

Comment:
Minor: IMO too long of a title for this code action:
```suggestion
    title: `Create a plugin configuration folder for this file.`,
```

Reply:
Yes, that also crossed my mind. Thanks!